src/js/apps/patients/patient/sidebar/sidebar_app.js

```diff
@@ -21,12 +20,9 @@ export default App.extend({
   },
   beforeStart({ patient }) {
     const workspacePatient = Radio.request('entities', 'fetch:workspacePatients:byPatient', patient.id);
-    const fields = map(Radio.request('widgets', 'sidebarWidgets:fields'), fieldName => {
```

Comment:
we can probably remove this from the widgets service

Reply:
Removed 👍